### ISSUE TITLE
Relocate and make progress-bar show on Safari iOS

### ISSUE THREAD

**Issue by ellocke** [ellocke/GithubPage#8]
i.e. with jQuery:

`$('.progress-bar-container').detach().appendTo('.mobile-navbar');`
